performant-software/Annotation-Studio
# Fix broken pasting in annotation editor

# What this PR does

This PR simply recompiles the assets for COVE, which include the TinyMCE plugin responsible for the annotation editing field. It's still not clear what broke it, but recompiling fixes the bug, as you can see at https://staging.covecollective.org

# How to test

1. Open any annotatable document
2. Double-click a word to bring up the annotation button
![Screen Shot 2022-04-06 at 12 59 22 PM](https://user-images.githubusercontent.com/64725469/162028242-455a4c62-bf2a-4ad9-9b4c-32371cacd3a2.png)
3. Click the button, opening the annotation editor
![Screen Shot 2022-04-06 at 11 51 25 AM](https://user-images.githubusercontent.com/64725469/162028340-b5bf3c41-3b1c-4367-9416-46536a623020.png)
4. Paste in some plain text
5. The text should appear in the box, and no red errors should appear in your browser console
